src/Image/ImageService.php

```diff
@@ -32,10 +32,23 @@ final class ImageService extends AbstractService
      */
     public function getImageLink(int $imageId, int $width = null, int $height = null): UriInterface
     {
-        $query = http_build_query(array_filter(['w' => $width, 'h' => $height]));
+        static $cache = [];
 
-        $apiBaseUrl = rtrim(to_string($this->client->getBaseUri()), '/');
+        $uid = "id$imageId.w$width.h$height";
 
-        return new Uri("{$apiBaseUrl}/image/${imageId}?${query}");
+        if (!isset($cache[$uid])) {
```

Comment:
```suggestion
        if (false === array_key_exists($cache[$uid])) {
```

Reply:
je ne pense pas que j'ai besoin de ce niveau de vérification dans mon cas de figure si
https://github.com/wizaplace/wizaplace-php-sdk/blob/eb66d2d85ea0cfb17bc84e44f3ce8817ff89d7f6/src/Image/ImageService.php
me retourne null j'ai bien envie de ne pas réitéré le process suivant :